### ISSUE TITLE
(Feature Request) FreeBSD support

### ISSUE THREAD

**Issue by hardBSDk** [AtomsDevs/Atoms#41]
There's a way to make it work on FreeBSD? because FreeBSD has it's own sandbox using namespaces called jails.

(It offers Linux binary compatibility too).

**Comment by orowith2os** [comment]
I don't think this is possible *at all* unless you have a full (be it a compat layer or an actual) Linux kernel above the containers. You could probably make this manage *just* FreeBSD envs, but I don't think that's necessary.

The Linux binary compatibility tool probably isn't enough for this use case.

**Comment by mirkobrombin** [comment]
Thanks for the idea. The official cpak provider and Flatpak frontend require Linux kernel interfaces, so FreeBSD support is outside the scope of Atoms 2.
